src/test/java/uk/nhs/hee/tis/trainee/forms/api/FormRPartAResourceTest.java

```diff
@@ -130,7 +135,7 @@ void postShouldNotCreateFormWhenNoToken() throws Exception {
     mockMvc.perform(post("/api/formr-parta")
             .contentType(TestUtil.APPLICATION_JSON_UTF8)
             .content(TestUtil.convertObjectToJsonBytes(dto)))
-        .andExpect(status().isBadRequest());
+        .andExpect(status().isForbidden());
```

Comment:
The change in response here and in other tests is due to the interceptor response. I _think_ this is not a (serious) breaking change.

Reply:
It's a more accurate response, so I say this is fine. Can't foresee it breaking anything, unless a third party is using our undocumented API 😉